src/main/java/io/javalin/Javalin.java

```diff
@@ -202,6 +200,33 @@ public Javalin defaultCharacterEncoding(String characterEncoding) {
         return this;
     }
 
+    /**
```

Comment:
No javadoc for now. It's better to merge them all at the same time from your other change.